website/docs/docs/ai/using.md

```diff
@@ -414,4 +414,44 @@ In AI, context is key. The more relevant information you can provide the more ta
 
 ### 3. Use Test Templates
 
-Test Templates reduce the amount of refactoring required after generation. If you start to see a common generation pattern you wish to avoid, update the Test Template to help PactFlow AI generate improved output.
\ No newline at end of file
+Test Templates reduce the amount of refactoring required after generation. If you start to see a common generation pattern you wish to avoid, update the Test Template to help PactFlow AI generate improved output.
+
+## Comparing PactFlow AI with Other AI Tools
```

Comment:
Looking at the site nav, might this live better in the intro, rather than under using, or should we link in the ai intro page, looking to see how PactFlow AI compares? Read here - link